specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -2313,6 +2630,173 @@
         }
       },
       "description": "The result returned from a check name availability request."
+    },
+    "AttachedDatabaseConfigurationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AttachedDatabaseConfiguration"
+          },
+          "description": "The list of attached database configurations."
+        }
+      },
+      "description": "The list attached database configurations operation response."
+    },
+    "AttachedDatabaseConfiguration": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of the attached database configuration.",
+          "$ref": "#/definitions/AttachedDatabaseConfigurationProperties"
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Class representing an attached database configuration."
+    },
+    "AttachedDatabaseConfigurationProperties": {
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ResourceProvisioningState",
+          "description": "The provisioned state of the resource."
+        },
+        "databaseName": {
+          "type": "string",
+          "description": "The name of the database which you would like to attach, use * if you want to follow all current and future databases."
+        },
+        "clusterResourceId": {
+          "type": "string",
+          "description": "The resource id of the cluster where the databases you would like to attach reside."
+        },
+        "attachedDatabaseNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "The list of databases from the clusterResourceId which are currently attached to the cluster."
```

Comment:
What is the form of the entries in this list?  Are they resource ids, simple names?  A sample would be useful in the description.